packages/block-editor/src/store/private-selectors.js

```diff
@@ -71,35 +71,33 @@ export function getLastInsertedBlocksClientIds( state ) {
  * @return {BlockEditingMode} The block editing mode. One of `'disabled'`,
  *                            `'contentOnly'`, or `'default'`.
  */
-export const getBlockEditingMode = createRegistrySelector(
-	( select ) =>
-		( state, clientId = '' ) => {
-			const explicitEditingMode = getExplcitBlockEditingMode(
-				state,
-				clientId
-			);
-			const rootClientId = getBlockRootClientId( state, clientId );
-			const templateLock = getTemplateLock( state, rootClientId );
-			const name = getBlockName( state, clientId );
-			const isContent =
-				select( blocksStore ).__experimentalHasContentRoleAttribute(
-					name
-				);
-			if (
-				explicitEditingMode === 'disabled' ||
-				( templateLock === 'contentOnly' && ! isContent )
-			) {
-				return 'disabled';
-			}
-			if (
-				explicitEditingMode === 'contentOnly' ||
-				( templateLock === 'contentOnly' && isContent )
-			) {
-				return 'contentOnly';
-			}
-			return 'default';
-		}
-);
+export const getBlockEditingMode = ( state, clientId = '' ) => {
+	const explicitEditingMode = getExplcitBlockEditingMode( state, clientId );
```

Comment:
You probably want to fix a typo here, adding a missing _i_ to the selector name.